src/lib/Microsoft.Health.Common/Telemetry/MetricExtension.cs

```diff
@@ -15,6 +16,9 @@ public static class MetricExtension
 
         public static Metric CreateBaseMetric(this MetricDefinition iomtMetric, string category, string operation)
         {
+            EnsureArg.IsNotNullOrWhiteSpace(category, nameof(category));
```

Comment:
Also ensure that `iomtMetric` is not null; otherwise calling `ToString()` below on a null object throws a null reference exception?

Reply:
Unless someone explicitly calls null.CreateBaseMetric(), I don't see the need to check here.